app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java

```diff
@@ -0,0 +1,85 @@
+package com.pps.globant.fittracker.mvp.presenter;
+
+import android.app.Dialog;
+import android.webkit.WebView;
+
+import com.pps.globant.fittracker.MainActivity;
+import com.pps.globant.fittracker.mvp.model.InstagramLoginModel;
+import com.pps.globant.fittracker.mvp.view.IntagramLoginView;
+import com.pps.globant.fittracker.utils.BusProvider;
+import com.pps.globant.fittracker.utils.MyWVClient;
+import com.squareup.otto.Bus;
+import com.squareup.otto.Subscribe;
+
+public class InstagramLoginPresenter {
+    private final static String CODE = "code";
+    private final static String ERROR = "error";
+    private static final String EQUAL_SIGN = "=";
+    private final InstagramLoginModel model;
+    private final IntagramLoginView view;
+    public Dialog dialog;
+    public MainActivity activity;
+    public Bus bus;
+
+
+    public InstagramLoginPresenter(Bus bus) {
+        this.bus = bus;
+        BusProvider.register(this);
+        model = new InstagramLoginModel(bus);
+        view = new IntagramLoginView();
+    }
+
+    public void igButtonLoginClick(MainActivity activity) {
+        this.activity = activity;
+        dialog = new Dialog(activity);
+        WebView webView = new WebView(activity);
+        webView.setVerticalScrollBarEnabled(false);
+        webView.setHorizontalScrollBarEnabled(false);
+        webView.setWebViewClient(new MyWVClient(dialog, bus));
+        webView.getSettings().setJavaScriptEnabled(true);
+        webView.loadUrl(model.getAuthUrlFull());
+        dialog.setContentView(webView);
+    }
+
+
+    public void handleUrl(String url) {
```

Comment:
I dont understand the idea for that method

Reply:
I removed the method and put the code where the method was called. It was done in that way by an old implementation where was more complex, thanks